pootle/static/js/stats.js

```diff
@@ -84,6 +85,22 @@ const stats = {
       e.preventDefault();
       this.toggleChecks();
     });
+    $(document).on('click', '.js-toggle-more-checks', (e) => {
```

Comment:
In Chrome, when I click the _more..._ link, instead of expanding the list some elements are replaced by others. There seems to be something wrong in this method, although I'm afraid I haven't seen a clear pattern and I can't say exactly what :(